cmd/podman/machine/inspect.go

```diff
@@ -72,12 +72,19 @@ func inspect(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
+		podmanSocket, podmanPipe, err := provider.ConnectionInfo(name)
+		if err != nil {
+			return err
+		}
+
 		ii := machine.InspectInfo{
 			// TODO I dont think this is useful
```

Comment:
Drive-by comment: Podman 5.0 seems like the right time to address (or remove) the TODO.

Reply:
We agreed to change that to point at the actual config file, not the directory

Reply:
my memory was that we agreed to change the label so that it reflects Dir

Reply:
Fixed